bug_change_status_page.php

```diff
@@ -333,17 +333,18 @@
 		printf( '	<input type="hidden" name="resolution" value="%s" />' . "\n", config_get( 'bug_reopen_resolution' ) );
 	}
 ?>
-<?php if( access_has_bug_level( config_get( 'private_bugnote_threshold' ), $f_bug_id ) ) { ?>
+<?php
+	$t_default_bugnote_view_status = config_get( 'default_bugnote_view_status' );
+	$t_bugnote_private = $t_default_bugnote_view_status == VS_PRIVATE;
+	$t_bugnote_class = $t_bugnote_private ? 'form-control bugnote-private' : 'form-control';
```

Comment:
$t_bugnote_private is used just one time, so maybe you want to remove previous line
```
$t_bugnote_private = $t_default_bugnote_view_status == VS_PRIVATE;
```
and change current line to
```
$t_bugnote_class = $t_default_bugnote_view_status == VS_PRIVATE ? 'form-control bugnote-private' : 'form-control';
```
But feel also free to merge without this change.